Xamarin.Essentials/Types/LocationExtensions.uwp.cs

```diff
@@ -14,6 +14,7 @@ public static partial class LocationExtensions
                 Latitude = mapLocation.Point.Position.Latitude,
                 Longitude = mapLocation.Point.Position.Longitude,
                 Altitude = mapLocation.Point.Position.Altitude,
+                AltitudeReferenceSystem = mapLocation.Point.AltitudeReferenceSystem,
```

Comment:
You will need some type of converstion system. Such as https://github.com/xamarin/Essentials/blob/develop/Xamarin.Essentials/Types/SensorSpeed.uwp.cs

Reply:
Well, the enum `Xamarin.Essentials.AltitudeReferenceSystem` is an exact copy of `Windows.Devices.Geolocation.AltitudeReferenceSystem`, using the same enum values, therefore conversion should be necessary, right? I even thought about directly reusing the enum in XE without a copy, but since it's only available on Windows, I did not see a way to do that for all platforms.

Reply:
Ah, I see the build failure now. So yes, seems I'm missing a cast there (I did not get that error when I built in VS).

Reply:
Fixed in 1610891.